test/cases/attributes_test.rb

```diff
@@ -99,20 +99,20 @@ def test_joined_spatial_attribute
     private
 
     def create_foo
-      Foo.connection.create_table :foos, force: true do |t|
+      Foo.lease_connection.create_table :foos, force: true do |t|
```

Comment:
these test case work in isolation

Reply:
What do you mean?

Reply:
Thanks for act, I still don't get your first point though..